tutorials/claim-device.md

```diff
@@ -0,0 +1,11 @@
+The WebAccess DMP SysAdmins have the responsibility to add or create all devices in the plattform, so you can assume the plattform is aware and can identify all devices you have. So in order for you to prove the device is yours, you need to provide DMP all the identifiers of your device. This process is call claiming your device.
+What are these identifiers? Let's go to Devices and then click on the claim device icon. This is the claim device modal. The second, third and fourth fields are the three identifiers of a device. You need to complete them all with correct values to claim the device to your company. All devices will have a serial number and a mac address. The IMEI is an identifier only for devices with a cellular module. You should get this information on the box of your device, but if you have access to the web interface of the device like I have access now, we recommend you to install the DMP client if is not there yet and go to "Claim device on plattform". Here you can see all information together. Let's copy the values one by one to the claim modal. The alias is a field to add a name to identify the device. Otherwise you'll be shown the mac address. Let's add some identifier here. Now we need to select the company we want it to belong to. Let's select the first company, and click to claim. The device was connected already so you can see its status is connected and synchronised. From now on the device belongs to this company and all users with device permissions on that company will be able to interact with the device.
+Now, suppose after some time configuring the device, we need to allow the installers to connect the device in some location and do some configuration there. One way we can do this is by having a different company where these devices are put temporarily, the Installers company. For this we have to re-claim this device.
+First of all we need to release the device from our company, just by clicking on the release button. We are asked to confirm, and after that, the device is gone.
+Now we need to claim again. We need to click on claim, and complete the form with the device identifiers again. This time we will select the Installers company and we can add it. As you can see, the alias we had in the first company didn't make it to this Installers company. Is related to the company it belongs to. As you can see I can see the device again and the company changed. Now the installer we created on a previous video can see the device too.
+
+You may be thinking that this process can easily become cumbersome when you have to claim and re-claim many times. This is a limitation of the UI, as we didn't intend to develop an UI for all our customers use cases, but a general purpose one. For these cases we recommend you to use the API directly. We wrote a How to guide for a similar use case. Under this video you will find the link to it, [How to create devices from a list and claim them](/how-tos/create-and-claim-devices.md)
```

Comment:
remove sentance "this is a limitation... general purpose one."
then re-phrase the recommendation.

rule of thumb: NEVER talk about limitations, constraints, what's NOT there... talk about what IS there.